internal/cmd/operator/start_controller_manager.go

```diff
@@ -124,22 +123,22 @@ func (c *ControllerManagerCommand) run(cmd *cobra.Command, argv []string) error
 	}
 
 	// Restrict to the following namespaces - subject to change.
-	namespaces := [...]string{"default", "oran", "o2ims", "oran-o2ims"} // List of Namespaces
-	defaultNamespaces := make(map[string]cache.Config)
+	//namespaces := [...]string{"default", "oran", "o2ims", "oran-o2ims"} // List of Namespaces
+	//defaultNamespaces := make(map[string]cache.Config)
 
-	for _, ns := range namespaces {
-		defaultNamespaces[ns] = cache.Config{}
-	}
+	//for _, ns := range namespaces {
+	//	defaultNamespaces[ns] = cache.Config{}
+	//}
```

Comment:
@irinamihai - I assume this was done because we're using a new namespace for each cluster template. Is this going to be an issue? Do we need to look at what would be required to put the cluster templates all in one of the ims namespaces?

Reply:
> this was done because we're using a new namespace for each cluster template
Yes. And these lines were restricting the namespaces that are watched by the controllers.

It wasn't a requirement to have just these namespaces, so I think we can remove these lines completely in the future.